patches/api/0318-Add-Git-information-to-version-command-on-startup.patch

```diff
@@ -0,0 +1,164 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Professor Bloodstone <git@bloodstone.dev>
+Date: Sun, 20 Jun 2021 01:48:31 +0200
+Subject: [PATCH] Add Git information to version command/on startup
+
+
+diff --git a/src/main/java/io/papermc/paper/util/JarManifests.java b/src/main/java/io/papermc/paper/util/JarManifests.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..cb3d68b28f66a8559018bd19bc106d136f2fc2f4
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/util/JarManifests.java
```

Comment:
If I understand correctly, this should be enough attribution. But maybe I'm wrong?
Should it possibly be a standalone patch with original author?

Reply:
Yes, this is good.